packages/ui-views/templates/ProjectTemplate/projectTemplate.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react'
 import Sidebar from '../../organisms/Sidebar'
 import SecondaryMenu from '../../organisms/SecondaryMenu'
 import * as themeable from 'react-themeable'
-import BindAgent, { AgentType } from '@vflows/bindings/agent/agent'
+import BindAgent, { AgentType } from '@vflows/bindings/Agent/agent'
```

Comment:
these should be double quotes, and imports need semicolons

Reply:
Noted. This change was automatically done by WebStorm so I didn't manually inspect each change. I'll go ahead and refactor these files.